repository-management.md

```diff
@@ -0,0 +1,101 @@
+# Repository Management
+
+## Package Names
+
+Package names for the official `wp-cli` GitHub repository need to be discussed upfront with the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
+
+Package names for WP-CLI commands must end with the `-command` suffix.
+
+_Example: `wp-cli/scaffold-package-command`._
+
+## Package Versions
+
+Package version follow the [Semantic Versioning Specification (SemVer)](http://semver.org/).
+
+## Milestones
+
+Open milestones usually point to the next version to be released.
+
+As the release notes are built using the milestones, pull requests that are to be merged need to be set to the upcoming milestone, so that these pull requests will be part of the release notes.
```

Comment:
Both pull requests and issues addressed in a milestone should be assigned to the milestone.

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/b2a3d71cae8daf696a6e0269deaec353e35502d8